sandbox/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django>=3.2
+Django>=4.2,<5.2
 wagtail>=5.2
-django-debug-toolbar==3.2.2
+django-debug-toolbar==4.4.6
```

Comment:
The update here was because Django 5.1 needs a more recent version of debug toolbar